tests/unit/test_numpy_splitters.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+
+import pandas as pd
+import numpy as np
+import pytest
+
+from reco_utils.dataset.numpy_splitters import numpy_stratified_split
+
+
+@pytest.fixture(scope="module")
+def test_specs():
+
+    return {
+        "number_of_items": 50,
+        "number_of_users": 20,
+        "seed": 123,
+        "ratio": 0.6,
+        "tolerance": 0.01,
+        "fluctuation": 0.02,
+    }
+
+
+@pytest.fixture(scope="module")
+def python_dataset(test_specs):
+
+    """Generate a test user/item affinity Matrix"""
+
+    # fix the the random seed
+    np.random.seed(test_specs["seed"])
+
+    # generates the user/item affinity matrix. Ratings are from 1 to 5, with 0s denoting unrated items
+    X = np.random.randint(
+        low=0,
+        high=6,
+        size=(test_specs["number_of_users"], test_specs["number_of_items"]),
+    )
+
+    return X
+
+
+def test_random_stratified_splitter(test_specs, python_dataset):
```

Comment:
Why "random" and not "numpy"? (you are testing `numpy_stratified_split()`)

Reply:
right, the name referred to the previous nomenclature and I forgot to change it. Thanks for catching this